ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py

```diff
@@ -1593,3 +1593,26 @@ def test_torch_select(
         dim=axis,
         index=idx,
     )
+

```

Comment:
@sanju4725 You don't have any input strategies defined for the test using the `@handle_frontend_test` decorator? See other tests in this file for reference

Reply:
@sanju4725 I don't see any changes with respect to my comment above

Reply:
sorry sir,
 was unavailable for the past few days
 Ill get back right away